api/v1/models/AttendeeExtraInfo.js

```diff
@@ -1,6 +1,6 @@
 const Model = require('./Model');
 const AttendeeExtraInfo = Model.extend({
-  tableName: 'attendee_extra_infos',
+  tableName: 'attendee_longform',
```

Comment:
It doesn't make sense to change the table for a model that already exists, lets just create a new table and model for the new `long form` response

Reply:
Fixed in b7fb5a6707ba789a9ce989f74461bd975cd1807b